lib/nerves_network/dhclientv4.ex

```diff
@@ -17,7 +17,6 @@ defmodule Nerves.Network.Dhclientv4 do
   require Logger
   alias Nerves.Network.Utils
 
-  @renew 1
```

Comment:
are we removing this just because it's getting flagged in Dialyzer?

Reply:
Not sure about dialyser, but the elixir compiler said it was not being used. It is just old could I would think that was not fully refactored.